src/server/NetworkingServer.hpp

```diff
@@ -57,6 +59,7 @@ class NetworkingServer {
      ThreadSafeVector<GroupUpdate> mGroupUpdates;
      ThreadSafeVector<MineUpdate> mMineUpdates;
      ThreadSafeMap<sf::Uint32, sf::Vector2f> mClientMoves;
+     ThreadSafeMap<sf::Uint32, bool> mClientGroupable;
```

Comment:
What do you think about combining mClientGroupable and mClientMoves into one map that stores all the updates for a client

Reply:
not sure honestly. seems like it might be more complicated than simplifyng at this point

im thinking we need a better overall solution for synced state from the server

Im seeing a lot of similarities with web UI's and solutions like GraphQL. I dont feel ready yet to make changes till we're pretty convinced of the right way

Reply:
does your game dev book talk about this?

Reply:
ok lets leave it separate for now